allennlp/tests/data/dataset_readers/atis_test.py

```diff
@@ -0,0 +1,68 @@
+# pylint: disable=no-self-use,invalid-name
+from allennlp.data.dataset_readers import AtisDatasetReader
+from allennlp.common.testing import AllenNlpTestCase
+
+from allennlp.semparse.worlds import AtisWorld
+
+class TestAtisReader(AllenNlpTestCase):
+    def test_atis_read_from_file(self):
+        reader = AtisDatasetReader()
+        data_path = AllenNlpTestCase.FIXTURES_ROOT / "data" / "atis" / "sample.json"
+
+        instances = list(reader.read(str(data_path)))
+
+        assert len(instances) == 11
+        instance = instances[0]
+
+        assert instance.fields.keys() == \
+                {'utterance',
+                 'actions',
+                 'world',
+                 'target_action_sequence',
+                 'linking_scores'}
+
+        assert [t.text for t in instance.fields["utterance"].tokens] == \
+                ['show', 'me', 'the', 'one', 'way',
+                 'flights', 'from', 'detroit', 'to',
+                 'westchester', 'county']
+
+        assert isinstance(instance.fields['world'].as_tensor({}), AtisWorld)
+        # Check that the strings in the actions field has the same actions
+        # as the strings in the world.
+        valid_strs = set()
+        for prod in instance.fields['actions'].field_list:
```

Comment:
Avoid abbreviations.  Code is meant to be read, and it's way easier on the reader if you use a few more characters and actually write out full words.